src/encoded/static/components/dataset.js

```diff
@@ -3236,13 +3236,13 @@ const TreatmentConcentrationSeries = ({ context }, reactContext) => {
     // related datasets.
     const treatmentDurations = context.related_datasets.reduce((accTreatmentDurations, relatedDataset) => {
         // Collect any biosamples found in all related datasets.
-        const biosamples = relatedDataset.replicates.reduce((accBiosamples, replicate) => {
+        const biosamples = relatedDataset.replicates?.reduce((accBiosamples, replicate) => {
             const biosample = replicate.library && replicate.library.biosample;
             return biosample ? accBiosamples.concat(biosample) : accBiosamples;
         }, []);
 
         // Collect durations in the biosample treatments and compose them into displayable strings.
-        const collectedDurations = biosamples.reduce((accCollectedDurations, biosample) => {
+        const collectedDurations = biosamples?.reduce((accCollectedDurations, biosample) => {
```

Comment:
If `biosamples` is undefined then `collectedDurations` becomes undefined here, which would cause the return value from this function to be an array containing an undefined entry. I think this would fix that:
```
return [...new Set(collectedDurations ? accTreatmentDurations.concat(collectedDurations) : accTreatmentDurations))];
```

Reply:
Will implement this! Thanks @forresttanaka